src/Console/Commands/ModulesCache.php

```diff
@@ -15,25 +17,26 @@ class ModulesCache extends Command
 	
 	protected $description = 'Create a cache file for faster module loading';
 	
-	public function handle(ModuleRegistry $registry, Filesystem $filesystem)
+	public function handle(AutoDiscoveryHelper $helper, CacheHelper $cache)
 	{
 		$this->call(ModulesClear::class);
 		
-		$export = $registry->modules()
-			->map(function(ModuleConfig $module_config) {
-				return $module_config->toArray();
-			})
-			->toArray();
+		foreach ($helper->toArray() as $key => $value) {
+			$cache->set($key, $value);
+		}
```

Comment:
The separation between the cache helper and the auto discovery helper doesn't feel quite right yet.

Reply:
Small nitpick I noticed is that the cache file `bootstrap/cache/modules.php` has the file paths repeated in the keys and the values. 

```
  'migrations' =>
  array (
    '/Users/eric.junker/my-app/app-modules/meetings/database/migrations' => '/Users/eric.junker/my-app/app-modules/meetings/database/migrations',
  ),
```

If you want to remove that duplication I think you just need to change `FinderCollection::__construct()` to just `yield $value` instead of `yield $key => $value`